resources/lang/en/validation.php

```diff
@@ -95,6 +95,7 @@
     'url'                  => 'The :attribute format is invalid.',
     'unique_undeleted'     => 'The :attribute must be unique.',
     'non_circular'         => 'The :attribute must not create a circular reference.',
+    'not_array'            => 'The :attribute field can not be an array.',
```

Comment:
Change this string to use "cannot" and this looks great :)

Reply:
I don't know what you're talking about 😄

Reply:
:P

Reply:
Hey, I told you I was anal-retentive when you signed up for this gig :D

Reply:
😂